APolitcalApp/src/components/cards/CardFullPetitionReview.vue

```diff
@@ -29,7 +29,7 @@ const goalString = computed( () =>
 
 <template>
 
-    <div class="bg-white rounded-lg border border-border bg-light-highlight drop-shadow-sm w-[448px] overflow-y-auto">
+    <div class="bg-white rounded-lg border w-[92.5%] border-border bg-light-highlight drop-shadow-sm overflow-y-auto xs:w-[448px]">
```

Comment:
For future reference, the best practice for getting a responsive website would probably be to use the w-5 or anything like that since it uses rems which is just a calculation determined by the size of the screen. Once again doesn'ty really matter right now